app/helpers/checkout_helper.rb

```diff
@@ -19,7 +19,7 @@ def checkout_adjustments_for(order, opts = {})
     adjustments.reject! { |a| a.originator_type == 'EnterpriseFee' && a.source_type != 'Spree::LineItem' }
     unless exclude.include? :admin_and_handling
       adjustments << Spree::Adjustment.new(
-        label: I18n.t(:orders_form_admin), amount: enterprise_fee_adjustments.sum(:amount)
+        label: I18n.t(:orders_form_admin), amount: enterprise_fee_adjustments.map(&:amount).sum
```

Comment:
At some point we discussed this and we went for to_a.sum instead of map. Maybe not a big difference? 
It would be good to have a unit test for this bug.

Reply:
I also I checked this. I bet there's not much difference but the actual issue is that `enterprise_fee_adjustments` is an array instead of an AR::Relation at this point already and we rely on Ruby to do the DB's work :see_no_evil: